R/class.R

```diff
@@ -0,0 +1,43 @@
+inject_super <- function(fun) {
+  e <- new.env()
+  e$super <- function() {
+    bt <- reticulate::import_builtins()
+    self <- as.list(parent.frame())$self
+    bt$super(self$`__class__`, self)
+  }
+  environment(e$super) <- environment(fun)
+  environment(fun) <- e
+  fun
+}
+
+#' Create a python class
+#' 
+#' @param classname The classname
+#' @param defs The definitions 
+#' @param inherit List of python classes
```

Comment:
Does this accept Python classes by name? Or do they need to be actual Python class objects?

Reply:
They need to be Python class objects